sources/packages/forms/src/form-definitions/stepparentwaiverappeal.json

```diff
@@ -0,0 +1,197 @@
+{
+    "title": "Appeals - Step Parent Waiver",
+    "name": "stepParentWaiverAppeal",
+    "path": "stepparentwaiverappeal",
+    "type": "form",
+    "display": "form",
+    "tags": [
+        "common"
+    ],
+    "components": [
+        {
+            "input": true,
+            "tableView": true,
+            "key": "maxUploadedFiles",
+            "label": "maxUploadedFiles",
+            "persistent": false,
+            "type": "hidden",
+            "lockKey": true,
+            "calculateValue": "value = 25;",
+            "calculateServer": true
+        },
+        {
+            "label": "Parents",
+            "key": "parents",
+            "type": "hidden",
+            "input": true,
+            "tableView": false
+        },
+        {
+            "label": "HTML",
+            "tag": "h3",
+            "className": "category-header-medium primary-color",
+            "attrs": [
+                {
+                    "attr": "",
+                    "value": ""
+                }
+            ],
+            "content": "Step parent waiver appeal",
+            "refreshOnChange": false,
+            "key": "header",
+            "type": "htmlelement",
+            "input": false,
+            "tableView": false,
+            "hideLabel": true
+        },
+        {
+            "label": "HTML",
+            "tag": "p",
+            "attrs": [
+                {
+                    "attr": "",
+                    "value": ""
+                }
+            ],
+            "content": "Important: You can appeal to waive your step-parent’s fixed contribution if your parent has had a recent (within the past 5 years) marriage or common-law relationship with a step-parent, where the step-parent has not assumed financial responsibility for you and does not claim you as a dependent on their taxes.",
+            "refreshOnChange": false,
+            "key": "subHeader",
+            "type": "htmlelement",
+            "input": false,
+            "tableView": false,
+            "hideLabel": true,
+            "isNew": false
+        },
+        {
+            "label": "Please indicate which parent should be waived from the application",
+            "optionsLabelPosition": "right",
+            "inline": false,
+            "tableView": true,
+            "validate": {
+                "required": true,
+                "custom": "const allowedValues = !!data.parents\n  ? data.parents.map((parent) => parent.id)\n  : [];\n  \nvalid = allowedValues.includes(input) ? true : \"Invalid value for selected parent\";"
```

Comment:
We should have a custom validation here to enforce that two parents should be present for selection. Since the hidden fields do not allow validation, it would enforce the rule at the dry run level, keeping the form-specific validation inside the form.io itself.